x-pack/solutions/observability/plugins/slo/public/components/slo/slo_outdated_callout/index.tsx

```diff
@@ -48,19 +51,21 @@ export function SloOutdatedCallout() {
               values={{ total: data.total }}
             />
           </p>
-          <p>
-            <EuiButton
-              color="warning"
-              data-test-subj="o11ySloOutdatedCalloutViewOutdatedSloDefinitionsButton"
-              fill
-              onClick={handleClick}
-            >
-              <FormattedMessage
-                id="xpack.slo.outdatedSloCallout.buttonLabel"
-                defaultMessage="Review Outdated SLO Definitions"
-              />
-            </EuiButton>
-          </p>
+          {!!experimentalFeatures?.management.enabled && (
+            <p>
+              <EuiButton
+                color="warning"
+                data-test-subj="o11ySloOutdatedCalloutViewOutdatedSloDefinitionsButton"
+                fill
+                onClick={handleClick}
+              >
+                <FormattedMessage
+                  id="xpack.slo.outdatedSloCallout.buttonLabel"
+                  defaultMessage="Review Outdated SLO Definitions"
+                />
+              </EuiButton>
+            </p>
+          )}
```

Comment:
you can just remove the flag check here since we need to display this in case of outdated SLOs